src/sentry/api/endpoints/team_all_unresolved_issues.py

```diff
@@ -80,19 +65,52 @@ def calculate_unresolved_counts(
         .annotate(open=Count("id"))
     )
 
+    # Pull extra data to do deduplication in Python. (Inefficient to do in SQL via subqueries
+    # (see ISWF-549); cannot do via DISTINCT ON state because of Django limitations.)
     bucketed_issues = (
         GroupHistory.objects.filter_to_team(team)
         .filter(group_history_environment_filter, date_added__gte=start, date_added__lte=end)
-        .annotate(bucket=TruncDay("date_added"), prev_status=prev_status_sub_qs)
-        .filter(dedupe_status_filter)
-        .order_by("bucket")
-        .values("project", "bucket")
         .annotate(
-            open=Count("id", filter=Q(status__in=OPEN_STATUSES)),
-            closed=Count("id", filter=Q(status__in=CLOSED_STATUSES)),
+            bucket=TruncDay("date_added"),
+            state=Case(
+                When(status__in=OPEN_STATUSES, then=Value("open")),
+                When(status__in=CLOSED_STATUSES, then=Value("closed")),
+                default=Value("other"),
+            ),
+        )
+        .order_by(
+            "group_id",
+            "status",
+            "bucket",
         )
+        .distinct("group_id", "status")
+        .values("project", "group_id", "bucket", "state")
     )
 
+    # We only want one open & one close for each group_id
+    processed_groups = {"open": set(), "closed": set()}
```

Comment:
If a group was opened/closed multiple times over several day this will filter it out I think? Should we do this in a progressive way, where we store the last state we saw for a group, and increment when the state changes?